webrtc/videoroom_advanced/.gitignore

```diff
@@ -1,6 +1,7 @@
 .idea/
 *.iml
 priv/certs/*.pem
+priv/tls_cert.pem
```

Comment:
Can't we store our cert in `priv/certs/`? If no, then I would call this like `integrated_turn_cert_pem`